packages/react-components/src/templates/ProfileOutputs.tsx

```diff
@@ -0,0 +1,102 @@
+import { ResearchOutputPublishingEntities } from '@asap-hub/model';
+import { css } from '@emotion/react';
+import React, { ComponentProps } from 'react';
+import { SharedResearchList } from '.';
+import { LinkConditional } from '../atoms';
+import { createMailTo } from '../mail';
+import { perRem } from '../pixels';
+import NoOutputsPage from './NoOutputsPage';
+
+const containerStyles = css({
+  padding: `${36 / perRem}em 0`,
+});
+
+export type ProfileOutputsProps = Omit<
+  ComponentProps<typeof SharedResearchList>,
+  'children'
+> & {
+  userAssociationMember: boolean;
+  hasOutputs: boolean;
+  contactEmail?: string;
+  publishingEntity: ResearchOutputPublishingEntities;
+};
+
+const ProfileOutputs: React.FC<ProfileOutputsProps> = ({
+  researchOutputs,
+  numberOfItems,
+  numberOfPages,
+  currentPageIndex,
+  renderPageHref,
+  isListView,
+  cardViewHref,
+  exportResults,
+  listViewHref,
+  hasOutputs,
+  userAssociationMember,
+  contactEmail,
+  publishingEntity,
+}) => (
+  <div css={containerStyles}>
+    {hasOutputs ? (
+      <SharedResearchList
+        exportResults={exportResults}
+        researchOutputs={researchOutputs}
+        numberOfItems={numberOfItems}
+        numberOfPages={numberOfPages}
+        currentPageIndex={currentPageIndex}
+        renderPageHref={renderPageHref}
+        isListView={isListView}
+        cardViewHref={cardViewHref}
+        listViewHref={listViewHref}
+      />
+    ) : userAssociationMember ? (
+      <NoOutputsPage
```

Comment:
This feels like allot of duplication. I think it would be better to use two ternary's in the props over duplicating the entire description

Reply:
This is the same as above, I've just change the name of ownTeam to userAssociationMember in order to be more general.

Reply:
My issue is that your duplicating the entire description twice for a couple word changes. It would be less duplication by just including two ternary operators within the description and title. Less duplication = better